tests/bgen/bgen-tests.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
     <PackageReference Include="Mono.Cecil" Version="0.11.4" />
     <PackageReference Include="Moq" Version="4.18.4" />
-    <PackageReference Include="MSBuild.StructuredLogger" Version="2.1.846" />
+    <PackageReference Include="MSBuild.StructuredLogger" Version="2.2.2" />
```

Comment:
Instead of having to change these values in so many places, wouldn't it make sense to use a MSBuild var or centralized package versions? https://learn.microsoft.com/en-us/nuget/consume-packages/Central-Package-Management

Reply:
Good idea; I ended up doing it slightly different by just creating a property with the version in Directory.Build.props instead.